Sources/_CShims/include/io_shims.h

```diff
@@ -75,5 +77,13 @@ typedef struct _REPARSE_DATA_BUFFER {
     };
 } REPARSE_DATA_BUFFER;
 
+static inline intptr_t GetSymbolicLinkReparseBufferPathBufferOffset(void) {
```

Comment:
For the other shims, we've been prefixing functions with something like `_ioshims_` should we follow suit here (@parkera I can't remember if that's required for some reason or just styling. Since these are `static inline` I don't think they'd have symbols so perhaps it doesn't matter for these?)

Reply:
A styling choice to make it obvious where we are using them (and hopefully can get rid of it in the future).

Reply:
I don't see these as being possible to remove anytime soon - there is no equivalent of `offsetof`.

Reply:
Does `MemoryLayout<REPARSE_DATA_BUFFER>.offset(of: \.SymbolicLinkReparseBuffer.PathBuffer)` work here instead of the c shim?

Reply:
It does not, we end up with `nil` for the offset (the type is not exactly the easiest on the clang importer).

Reply:
Gotcha, in that case could you update the naming of these functions to follow the naming convention of the other cshim functions to try to keep them relatively consistent? Something along the lines of `_ioshims_symboliclinkreparsebuffer_pathbufferoffset` maybe?